contrib/pkg/createcluster/create.go

```diff
@@ -3,18 +3,19 @@ package createcluster
 import (
 	"encoding/json"
 	"fmt"
-	awsutils "github.com/openshift/hive/contrib/pkg/utils/aws"
-	gcputils "github.com/openshift/hive/contrib/pkg/utils/gcp"
-	openstackutils "github.com/openshift/hive/contrib/pkg/utils/openstack"
-	"github.com/openshift/hive/pkg/clusterresource"
-	"github.com/openshift/hive/pkg/gcpclient"
 	"io/ioutil"
 	"net/http"
 	"os"
 	"os/user"
 	"path/filepath"
 	"strings"
 
+	awsutils "github.com/openshift/hive/contrib/pkg/utils/aws"
```

Comment:
Move these imports down to the last section of imports, with the other "github.com/openshift/hive" imports.